generators/entity/index.js

```diff
@@ -57,6 +58,23 @@ module.exports = class extends EntityGenerator {
     get configuring() {
         const phaseFromJHipster = super._configuring();
         const jhipsterNetPhaseSteps = {
+            loadConfig() {
+                // Update current context with config from file.
+                Object.assign(this.context, this.entityStorage.getAll());
+                loadRequiredConfigIntoEntity(this.context, this.jhipsterConfig);
+            },
+            prepareForTemplates() {
+                const entity = this.context;
+                prepareEntityForTemplates(entity, this);
+
+                this.context.fields.forEach(field => {
+                    prepareFieldForTemplates(entity, field, this);
+                });
+
+                this.context.relationships.forEach(relationship => {
+                    prepareRelationshipForTemplates(entity, relationship, this);
+                });
+            },
             loadInMemoryDataNetBlueprint() {
```

Comment:
There is a lot happening in `loadInMemoryDataNetBlueprint`.
It belongs to `preparing` priority.

Very few changes seems related to client. I would recommend to move everything possible to entity-server and entity-client for simplicity.